src/main/java/frc/robot/util/PathPlannerCommands.java

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.util;
+
+import com.pathplanner.lib.commands.FollowPathHolonomic;
+import com.pathplanner.lib.path.PathPlannerPath;
+import com.pathplanner.lib.util.HolonomicPathFollowerConfig;
+import com.pathplanner.lib.util.ReplanningConfig;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystem.DriveSubsystem;
+import frc.robot.subsystem.PositioningSubsystem;
+
+public class PathPlannerCommands {
+    private static final ReplanningConfig REPLANNING_CONFIG = new ReplanningConfig();
+
+    public static Command getFollowPathCommand(
+            DriveSubsystem drive,
+            PositioningSubsystem positioning,
+            PathPlannerPath path,
+            boolean shouldAssumeRobotIsAtStart) {
+        // Manually flip the path ourselves, since we need the starting pose
+        if (DriverStation.getAlliance().map(a -> a == Alliance.Red).orElse(false)) {
+            path = path.flipPath();
+        }
+
+        Pose2d startPose = path.getStartingDifferentialPose();
```

Comment:
This should be getting the holonomic pose, not the differential pose. Differential is west-coast, but we have a swerve drive, which is holonomic.